docs/content/latest/tools/superset.md

```diff
@@ -0,0 +1,53 @@
+---
+title: Use Apache Superset with YugabyteDB YSQL
+headerTitle: Apache Superset
+linkTitle: Apache Superset
+description: Use Apache Superset to explore and visulize data in YugabyteDB.
+menu:
+  latest:
+    identifier: superset
+    parent: tools
+    weight: 2794
+isTocNested: true
+showAsideToc: true
+---
+
+[Apache Superset](https://superset.apache.org/) is a fast, lightweight and intuitive open-source data exploration and visualization tool that helps you query your data stored in YugabyteDB and visualize it using simple line charts to highly detailed geospatial charts.
+
+![Superset Dashboard](/images/develop/tools/superset/dashboard.png)
+
+## Before you begin
+
+Your YugabyteDB cluster should be up and running. If you're new to YugabyteDB, create a local cluster in less than five minutes following the steps in [Quick Start](../../quick-start/install). You can also get started with the free tier of [YugabyteDB Fully-Managed Cloud](https://www.yugabyte.com/cloud/). You also need to load a relevant database in your YugabyteDB for exploration and visualizaation. If using yugabyted, you can load the Northwind sample database with the ./bin/yugabyted demo connect command from your shell or you can follow [instructions here](https://docs.yugabyte.com/latest/sample-data/northwind/).
+
+You also need to install Apache Superset to explore and visualize your data. You can install Superset using [Docker Compose](https://superset.apache.org/docs/installation/installing-superset-using-docker-compose) (recommended) or from scratch using [python install (PIP)](https://superset.apache.org/docs/installation/installing-superset-from-scratch).
```

Comment:
What version of Python is required for this? If it's 3, as I suspect, change `(pip)` to `(pip3)` as a little clue for the reader.

```suggestion
You also need to install Apache Superset to explore and visualize your data. You can install Superset using [Docker Compose](https://superset.apache.org/docs/installation/installing-superset-using-docker-compose) (recommended) or from scratch using [Python (pip)](https://superset.apache.org/docs/installation/installing-superset-from-scratch).
```